Icon/IconGenerator.swift

```diff
@@ -191,8 +208,10 @@ extension View {
 
     /// Generates an image from the current View
     /// - Parameter size: The size of the image to generate
-    func generateImageData(size: CGSize) throws -> Data {
-        let wrapper = NSHostingView(rootView: self)
+    func generateImageData(size: CGSize, frame: Bool) throws -> Data {
+        let wrapper = frame
+            ? NSHostingView(rootView: self.frameIcon(dimension: min(size.width, size.height)))
```

Comment:
💯 Very clever to use `frameIcon(dimension:)` for this!

Reply:
Thx!